integration-tests/gatsby-source-wordpress/docker/wp-cli/start.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+echo "define( 'GRAPHQL_DEBUG', true );" >> /var/www/html/wp-config.php
```

Comment:
Put WPGraphQL in debugging mode so there are more helpful errors when tests fail